src/style.css

```diff
@@ -1 +1,56 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Consider adding a fallback font-family for your text content. Even though this rule does not directly set the font, it's a good practice to define a font-family higher up in your CSS to ensure text is rendered with a default font in case your chosen font fails to load.